keyboards/wsk/tkl30/config.h

```diff
@@ -0,0 +1,59 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
```

Comment:
We're trying to get all these vendor/product strings unique.

```suggestion
#define PRODUCT_ID      0x0B7F
```